xbmc/interfaces/legacy/Control.cpp

```diff
@@ -150,7 +150,7 @@ namespace XBMCAddon
     {
       if (!pGUIControl) return NULL;
 
-      LOCKGUI;
+      XBMCAddonUtils::GuiLock(languageHook);
```

Comment:
You are right `languageHook` is correct.
Looked at it again and replacing `XBMCAddonUtils::GuiLock(languageHook);` with `XBMCAddonUtils::GuiLock::GuiLock(languageHook);` everywhere worked for me on windows.